converter.py

```diff
@@ -20,23 +23,55 @@ def find_files(directory, pattern):
                 yield filename
 
 
+def get_dump_name(filename, extension):
+    if extension is None:
+        return filename
+    else:
+        split_name = os.path.basename(filename).split('.')
+        if len(split_name) > 1:
+            new_basename = '.'.join(split_name[:-1]) + extension
+        else:
+            new_basename = split_name[0] + extension
+        return os.path.join(os.path.dirname(fname), new_basename)
+
+
 logger = Logger('Converter')
 BASE_DIR = '/projects'
 paths = Paths()
+PHENOTYPE_PARAMS = (
+    "median_kernel_size",
+    "gauss_filter_sigma",
+    "linear_regression_size",
+    "phenotypes_inclusion",
+    "no_growth_monotonicity_threshold",
+    "no_growth_pop_doublings_threshold",
+)
 
-for file_type, pattern, loader, processor in (
+for file_type, pattern, loader, processor, new_ext in (
     (
         'project compilations',
         paths.project_compilation_pattern.format('*'),
         CompileImageAnalysisFactory.serializer.load,
         lambda data: sorted(data, key=lambda e: e.image.index),
+        None,
     ),
     (
         'project compilation instructions',
         paths.project_compilation_instructions_pattern.format('*'),
         CompileProjectFactory.serializer.load_first,
         None,
+        None,
     ),
+    (
+        'phenotypes extraction parameters',
+        paths.phenotypes_extraction_params,
+        np.load,
+        lambda data: {
+            PHENOTYPE_PARAMS[i]: p
```

Comment:
Have you considered not just using `p` directly here but for the `Trusted` et.al. convert it to the actual enum before sending it to jsonizer so that it's not just the floating enum value that is stored but information about what the Enum is?

Reply:
In this case only one of the values (the 4th one, which is optional, it can be as few as three) is an enum, the others are numbers. I could add a type conversion to the list though. Should be easy.

Reply:
Fixed